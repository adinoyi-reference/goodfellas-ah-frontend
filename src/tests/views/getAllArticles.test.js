import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AllArticlePage from '../../views/AllArticles';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<AllArticlePage />);
});

describe('ForgotPasswordPageUI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
