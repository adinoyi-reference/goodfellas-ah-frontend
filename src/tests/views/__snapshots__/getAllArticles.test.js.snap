// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordPageUI render features view should render as expected 1`] = `
<div>
  This is all articles page
</div>
`;
